docs/docs/reference/built-in-components/gatsby-plugin-image.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Gatsby Image plugin
+---
+
+This guide will show you how to configure your images, including choosing layouts, placeholders and image processing options. If you would like to learn how to set up the image plugins and use images on your site, see [the how-to guide](/docs/how-to/images-and-media/using-gatsby-plugin-image).
+
+## Components
+
+The Gatsby Image plugin includes two components to display responsive images on your site, used for static and dynamic images.
+
+- **[`StaticImage`](#staticimage):** Use this if the image is the same every time the component is used. _Examples: site logo, index page hero image_
+- **[`GatsbyImage`](#gatsbyimage):** Use this if the image is passed into the component as a prop, or otherwise changes. _Examples: Blog post hero image, author avatar_
+
+The following are props that can be passed to the components.
+
+### Shared props
+
+The following props can be passed to both `GatsbyImage` and `StaticImage`. You may also use any valid [`<img>` tag props](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attributes), which are forwarded to the underlying `<img>` element.
+
+| Prop              | Type                                                                        | Default       | Description                                                                                                                                     |
+| ----------------- | --------------------------------------------------------------------------- | ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| `alt` (Required)  | `string`                                                                    |               | Alternative text, passed to the `<img>` tag. Required for accessibility.                                                                        |
+| `as`              | `ElementType`                                                               | `"div"`       | The HTML element used for the outer wrapper.                                                                                                    |
+| `loading`         | `"eager" \| "lazy"`                                                         | `"lazy"`      | Loading behavior for the image. You should set this to `"eager"` for above-the-fold images to ensure they start loading before React hydration. |
```

Comment:
I think the escaping of the pipe doesn't work with MDX on .com, so while GitHub is fine, the website will show the backslash. I've used the mathematical divisor in the past -- see "Documentation" of this PR https://github.com/gatsbyjs/gatsby/pull/28485

Reply:
Thanks! It doesn't work in Typora either. Great fix.